financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -144,9 +155,9 @@ internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity(), Ma
         initialPane: Pane,
         reducedBranding: Boolean
     ) {
-        val context = LocalContext.current
+        val context = LocalContext.current as AppCompatActivity
```

Comment:
We have a few Compose utils in `payments-core` to retrieve the current activity in Compose. Do you want to move them to `stripe-core` and use those?

Reply:
https://github.com/stripe/stripe-android/pull/7081/commits/f42ca75bc383c0550a23ca72e998578183155dbb